docs/source/getting_started/active_learning.rst

```diff
@@ -199,13 +212,27 @@ boxes.
 
 Currently, the following scorers are available:
 
-- **object-frequency** uses model predictions to focus more on images which 
-  have many objects in them. Use this scorer if you want scenes with lots of 
-  objects in them like we usually want in computer vision tasks such as 
-  perception in autonomous driving.
-
-- **prediction-margin** uses the margin between 1.0 and the mean of the highest 
-  confidence prediction. Use this scorer to select images where the model is insecure.
+- **object-frequency**
+  This score measures the number of objects in the image. Use this scorer if
+  you want scenes with lots of objects in them. This is suited for computer vision
+  tasks such as perception in autonomous driving.
+
+- **prediction-margin**
+  This score is 1 - the mean of the highest confidence prediction. Use this scorer
+  to select images where the model is insecure about both whether it found an object
+  at all and the class of the object.
+
+- **classification_scores**
```

Comment:
Wouldn't it make sense to postpone this? The goal was to simplify the setup and make it more transparent how the scorers work. This is a new scorer. I think it's a great idea but maybe it makes more sense to first work on the ones we have?